netbox/templates/extras/configrevision.html

```diff
@@ -0,0 +1,213 @@
+  {# Clone/Edit/Delete Buttons #}
+  <div class="controls">
+    <div class="control-group">
+      {% plugin_buttons object %}
+
+      {# Extra buttons #}
+      {% block extra_controls %}{% endblock %}
+
+      {% if request.user|can_change:object %}
+        {% edit_button object %}
+      {% endif %}
+
+    </div>
+    <div class="control-group">
+      {% custom_links object %}
+    </div>
+  </div>
+{% endblock controls %}
+
+
+{% block tabs %}
+{% endblock %}
+
+{% block content %}
+  <div class="row">
+    <div class="col col-md-6">
+      <div class="card">
+        <h5 class="card-header">Rack Elevation</h5>
+        <div class="card-body">
+          <table class="table table-hover attr-table">
+            <tr>
+              <th scope="row">Rack elevation default unit height:</th>
+              <td>{{ object.data.RACK_ELEVATION_DEFAULT_UNIT_HEIGHT }}</td>
+            </tr>
+            <tr>
+              <th scope="row">Rack elevation default unit width:</th>
+              <td>{{ object.data.RACK_ELEVATION_DEFAULT_UNIT_WIDTH }}</td>
+            </tr>
+          </table>
+        </div>
+      </div>
+
+      <div class="card">
+        <h5 class="card-header">Power</h5>
+        <div class="card-body">
+          <table class="table table-hover attr-table">
+            <tr>
+              <th scope="row">Powerfeed default voltage:</th>
+              <td>{{ object.data.POWERFEED_DEFAULT_VOLTAGE }}</td>
+            </tr>
+            <tr>
+              <th scope="row">Powerfeed default amperage:</th>
+              <td>{{ object.data.POWERFEED_DEFAULT_AMPERAGE }}</td>
+            </tr>
+            <tr>
+              <th scope="row">Powerfeed default max utilization:</th>
+              <td>{{ object.data.POWERFEED_DEFAULT_MAX_UTILIZATION }}</td>
+            </tr>
+          </table>
+        </div>
+      </div>
+
+      <div class="card">
+        <h5 class="card-header">IPAM</h5>
+        <div class="card-body">
+          <table class="table table-hover attr-table">
+            <tr>
+              <th scope="row">IPAM enforce global unique:</th>
+              <td>{{ object.data.ENFORCE_GLOBAL_UNIQUE }}</td>
+            </tr>
+            <tr>
+              <th scope="row">IPAM prefer IPV4:</th>
+              <td>{{ object.data.PREFER_IPV4 }}</td>
+            </tr>
+          </table>
+        </div>
+      </div>
+
+      <div class="card">
+        <h5 class="card-header">Security</h5>
+        <div class="card-body">
+          <table class="table table-hover attr-table">
+            <tr>
+              <th scope="row">Allowed URL schemes:</th>
+              <td>{{ object.data.ALLOWED_URL_SCHEMES }}</td>
+            </tr>
+          </table>
+        </div>
+      </div>
+
+      <div class="card">
+        <h5 class="card-header">Banners</h5>
+        <div class="card-body">
+          <table class="table table-hover attr-table">
+            <tr>
+              <th scope="row">Banner login:</th>
```

Comment:
We can rename some of these parameters to be more human-friendly:

```suggestion
              <th scope="row">Login banner:</th>
```